bgfx.idl

```diff
@@ -438,6 +438,19 @@ struct.Attachment
 	.layer   "uint16_t"      --- Cubemap side or depth layer/slice.
 	.resolve "uint8_t"       --- Resolve flags. See: `BGFX_RESOLVE_*`
 
+--- Init attachment.
+func.Attachment.init
+	"void"
+	.handle "TextureHandle" --- Render target texture handle.
+	.access "Access::Enum"  --- Access. See `Access::Enum`.
+--	 { default = Access::Write }
```

Comment:
`default = "Access::Write" `

Reply:
Ah ok, I was basing it on `default = INT32_MAX` that doesn't have quotes.